packages/flutter/test/widgets/editable_text_shortcuts_tests.dart

```diff
@@ -69,6 +69,7 @@ void main() {
         child: SizedBox(
```

Comment:
unrelated to this PR, but maybe you can fix it nonetheless: I just notices that this fail is named incorrectly. It should end in `_test.dart` instead of `_tests.dart`. Could you maybe rename it?

Reply:
(I wonder if that naming problem actually means that this test doesn't run as part of our CI)

Reply:
Right, this would explain why this wasn't caught before. As far as I can tell this should have started failing after #97446 (before that TextPainter's line metrics cache wasn't cleared and as a result vertical runs could be jumping to incorrect offsets).

It appears to be the only file with "_tests" instead of "_test" in the name:

```
% find . -name "*_tests.dart"
./test/widgets/editable_text_shortcuts_tests.dart
```

Let me change it and update the PR.

Reply:
Another interesting fact is that since vertical runs rely on line metrics cache, they won't work in EditableText widgets that don't have fixed width+height (each vertical run will be invalidated on widget update right after the caret moves).